src/printer.js

```diff
@@ -312,6 +315,13 @@ function printArgumentsList(path, options, print, argumentsKey = "arguments") {
   return conditionalGroup(formsToConsider);
 }
 
+function wrapPropertyLookup(node, doc) {
+  const addCurly =
```

Comment:
should probably check that node is the right kind just to be safe, `node.offset.kind` could fatal if this is called incorrectly

Reply:
good point!